capstone-ui/src/components/Recommendations/Recommendations.jsx

```diff
@@ -17,6 +17,9 @@ export default function Recommendations({
   return (
     <div className="recommendations-component">
       <div className="recommendations-wrapper">
+        {recs != null && recs.length === 0 && (
+          <span>No recommended users.</span>
+        )}
         {recs &&
```

Comment:
recs != null

Reply:
I check for the condition that recs.length === 0 because if I only check for recs != null, then the text "No recommended users" would show along with the recommended users rendered in the following line.

Reply:
My suggestion is to update the first check recs && to recs != null && since it's easier to read that way. It's more explicit.